system/jlib/jscm.hpp

```diff
@@ -94,9 +93,11 @@ template <class CLASS> class Shared
 public:
     inline Shared()                              { ptr = NULL; }
     inline Shared(CLASS * _ptr, bool owned)      { ptr = _ptr; if (!owned && _ptr) _ptr->Link(); }
-    inline Shared(const Shared<CLASS> & other)   { ptr = other.getLink(); }
+    inline Shared(const Shared & other)          { ptr = other.getLink(); }
+#if defined(__cplusplus) and __cplusplus >= 201100
```

Comment:
This is because eclrtl.cpp disables c++11 support - which means when it includes this file it fails to compile.  (That was changed by HPCC-14564.)
It is likely to contain more problems in the long term - we may need to split that file in two with the absolute minimum compiled with c++11 disabled.